setup.cfg

```
@@ -0,0 +1,40 @@
+[coverage:run]
+include = 
+    src/api/*
+    src/backend/*
+    src/cms/*
+    gyz_api/*
```

```suggestion
    src/gyz_api/*
```
why is the base path missing here?